doc/source/user_guide/krylov.rst

```diff
@@ -45,23 +47,23 @@ to obtain the solution:
   method.
 
 
-Krylov method implementation in PyMADL
-======================================
+Krylov method implementation in PyMAPDL
+=======================================
 The PyMAPDL implementation of the Kylov method gives you customization
```

Comment:
```suggestion
The PyMAPDL implementation of the Krylov method gives you customization
```

Reply:
@kmkoshy  And look--I still missed the typo in Krylov, even after Joann pointed out the misspelling in yesterday's meeting!

Reply:
If I were paid every time I do a typo.... #ElonSecondRichestMan